tests/examples/valid.markdown

```diff
@@ -6,12 +6,12 @@ Based on VPAT® 2.4 edition 508/WCAG 2.0
 Drupal 9.1
 
 ## Report Date
-8/4/2021
+8/5/2021
 
 ## Product Description
 Content Management System
 
-## Contact Information
+## Author Information
```

Comment:
I'm tempted to leave this as Contact Information. In this case we are both the Vendor and the Author and we will want to have both ### Vendor & ### Author might be one way to resolve this?

Reply:
Good point. I can make that change.

Reply:
@mgifford updated.